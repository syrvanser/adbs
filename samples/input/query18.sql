SELECT * FROM Boats, Extra WHERE Extra.E >= Boats.E