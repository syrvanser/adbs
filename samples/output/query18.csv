101,2,3,2
101,2,3,3
101,2,3,4
101,2,3,5
101,2,3,6
102,3,4,3
102,3,4,4
102,3,4,5
102,3,4,6
103,1,1,1
103,1,1,2
103,1,1,3
103,1,1,4
103,1,1,5
103,1,1,6
107,2,8,2
107,2,8,3
107,2,8,4
107,2,8,5
107,2,8,6
