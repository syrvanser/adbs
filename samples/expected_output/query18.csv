103,1,1,1
101,2,3,2
103,1,1,2
107,2,8,2
101,2,3,3
102,3,4,3
103,1,1,3
107,2,8,3
101,2,3,4
102,3,4,4
103,1,1,4
107,2,8,4
101,2,3,5
102,3,4,5
103,1,1,5
107,2,8,5
101,2,3,6
102,3,4,6
103,1,1,6
107,2,8,6